### ISSUE TITLE
"there was a problem inviting users" when inviting new users.

### ISSUE THREAD

**Issue by sergey-vin** [Budibase/budibase#7771]
**Hosting**
- Cloud

**Describe the bug**
When I'm inviting users, I receive "there was a problem inviting users". Seems that it's because the same user was already invited to another tenant earlier.

**To Reproduce**
1. Go to BB main page -> Users -> Add user 
<img width="406" alt="image" src="https://user-images.githubusercontent.com/1080493/190162148-59888b5f-4b38-4914-9216-ec0d778ae22f.png">
2. Click "send email invites" 
<img width="413" alt="image" src="https://user-images.githubusercontent.com/1080493/190162266-f7d3a52b-3426-4f33-8799-b2a35be6545d.png">
3. See error
<img width="406" alt="image" src="https://user-images.githubusercontent.com/1080493/190162452-7897d74f-2c44-42c0-adfa-dc3841953540.png">

**Expected behavior**
User receives an invite via email

**Screenshots**
See above
  
**Additional context**
I've noticed 2 related things:
- if a user is already invited into account1 - it blocks adding of that user to account2. Removing a user from account1 makes it available to be added to account2.
- if user was invited to account2, but then removed - I can't add it to account2 again.
- If I add "+1" before "@" sign in an email, then I can invite the user, e.g. "example+1@gmail.com" instead of "example@gmail.com"

**Comment by sergey-vin** [comment]
hey! any news on that?

**Comment by shogunpurple** [comment]
Hi @sergey-vin - this is by design. Due to the multitenant nature of the cloud, we associate an email address with a tenant, and when they log in they are redirected to the correct tenant. We cannot have the same email address across different tenants.

You can get around this with exactly the same workaround you mentioned - by using email aliases such as `somemail+myalias@email.com`.

Closing for now as by design

**Comment by sergey-vin** [comment]
Thanks for reply. I was expecting something like that. At least it's not a bug.

What about self-hosted BB? Can I have 3 self-hosted apps, let's say one for dev, another for qa, another for prod, and share same users in all 3 of them?

Thanks.

**Comment by shogunpurple** [comment]
Yes, but you can do this in cloud as well - if the 3 apps are under the same budibase account and you assign the user permission to them they can access all 3.
